api/src/main/java/eu/coatrack/api/ServiceApi.java

```diff
@@ -75,17 +75,6 @@
     @Temporal(javax.persistence.TemporalType.DATE)
     private Date deletedWhen;
 
-    /*@OneToOne(optional = true)
```

Comment:
This part should stay for the time being, as we need to further discuss this feature.

Reply:
Done. The comment is restored.